resume_code/wos_api.py

```
@@ -0,0 +1,288 @@
+    all_records = []
+    first_record = 1
+    count = 100
+
+    while True:
+        params = {
+            "databaseId": "WOS",
+            "usrQuery": query,
+            "count": count,
+            "firstRecord": first_record,
+        }
+        resp = requests.get(WOS_API_BASE, headers=headers, params=params)
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)